src/controllers/hid/hidcontroller.h

```diff
@@ -112,8 +113,13 @@ class HidControllerJSProxy : public ControllerJSProxy {
         m_pHidController->sendReport(data, length, reportID);
     }
 
+	Q_INVOKABLE void sendFeatureReport(
+            const QList<int>& dataList, unsigned int reportID) {
```

Comment:
Please try using QByteArray again now that this is based on master.

Reply:
I tried again, but it doesn'tcompile:
[CXX] src\controllers\hid\hidcontroller.cpp
hidcontroller.cpp
src\controllers\hid\hidcontroller.cpp(305): error C2663: "QByteArray::prepend": für 5 Überladung(en) gibt es keine zulässige Konvertierung für den this-Zeiger
scons: *** [win64_build\src\controllers\hid\hidcontroller.obj] Error 2
scons: building terminated because of errors.
==============================

Building Mixxx failed.